e2e/cypress/integration/ui/settings/01_user_mgmt.spec.ts

```diff
@@ -26,81 +27,77 @@ describe('user management', () => {
     cy.get('app-user-table chef-button').contains('Create User').click();
     cy.get('app-user-management chef-modal').should('exist');
 
+    // adding a delay to more closely approximate an average human's type speed (200 char/min)
+    // the `should` will wait until the actual input value matches the expected value
+    // ref: https://github.com/cypress-io/cypress/issues/534
     cy.get('[formcontrolname=fullname]')
-      .type(name);
+      .type(name, { delay: 50 }).should('have.value', name);
```

Comment:
I would suggest either this...
```suggestion
      .type(name, TYPE_DELAY).should('have.value', name);
```
or at least this...
```suggestion
      .type(name, { delay: TYPE_DELAY }).should('have.value', name);
```
where you create a global constant of appropriate format rather than propagate that magic number in many places.

(Or, if it is not easy to do global constants with Cypress, at a minimum a file level constant.)

Reply:
Good point! I'm planning to turn this into a helper which should resolve the issue